### PR TITLE
More LHCb python3

### PR DESCRIPTION
**Pull request by mesmith75**

### CODE REVIEW

**Comment by alexanderrichards** [comment]
@mesmith75 ready to merge?

**Comment by mesmith75** [comment]
So I *think* this solves the monitoring issues.

**Comment by mesmith75** [comment]
@alexanderrichards I think this can be merged now

File: ganga/GangaCore/GPIDev/Adapters/IBackend.py [hunk 1]
```diff
@@ -494,12 +494,21 @@ def master_updateMonitoringInformation(jobs):
...
+                                if queues.totalNumIntThreads() < getConfig("Queues")['NumWorkerThreads']:
+                                    logger.info('done successfuly comparison')
+                                else:
+                                    logger.info('done succeful comparison 2')
+                            except Exception as err:
+                                logger.error('this is my error %s' % err)
                             if queues.totalNumIntThreads() < getConfig("Queues")['NumWorkerThreads']:
+                                print("about to add to queues")
                                 queues._addSystem(j.backend.updateMonitoringInformation, args=(subjobs_to_monitor,), name="Backend Monitor")
+                                print("added to uqueueue")
```
Comment (alexanderrichards) [inline ganga/GangaCore/GPIDev/Adapters/IBackend.py]: Can we lose the prints?

File: ganga/GangaCore/GPIDev/Adapters/IBackend.py [hunk 2]
```diff
@@ -494,12 +494,21 @@ def master_updateMonitoringInformation(jobs):
...
                         else:
                             j.backend.updateMonitoringInformation(subjobs_to_monitor)
                     except Exception as err:
-                        logger.error("Monitoring Error: %s" % err)
+                        logger.error("arse Monitoring Error: %s" % err)
```
Comment (alexanderrichards) [inline ganga/GangaCore/GPIDev/Adapters/IBackend.py]: :wink:

**Review by alexanderrichards — CHANGES_REQUESTED** [review]
Thanks for doing this @mesmith75 and good spot on the FunctionInput namedtuple rich comparison operators, I forgot about that.

**Review by alexanderrichards — APPROVED** [review]

**Pull request merged**